02-microarray/differential-expression_microarray_01_2-groups.html

```diff
@@ -3929,7 +3929,7 @@ <h2><span class="header-section-number">4.2</span> Import and set up data</h2>
 <div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a><span class="co"># Read in metadata TSV file</span></span>
 <span id="cb10-2"><a href="#cb10-2"></a>metadata &lt;-<span class="st"> </span>readr<span class="op">::</span><span class="kw">read_tsv</span>(metadata_file)</span></code></pre></div>
 <pre><code>## 
-## ── Column specification ──────────────────────────────────────────────
+## ── Column specification ──────────────────────────────────────────────────────────────────────────────────
```

Comment:
Why did it do this? It's a mystery, but should not affect final rendering.

Reply:
This is a mystery. I ran it with snakemake in the docker image, so I find this perplexing.